tests/lexer/tstrnumlits.nim

```diff
@@ -0,0 +1,157 @@
+# Test tkStrNumLit
+
+import std/macros
+import strutils
+
+macro assertAST(expected: string, struct: untyped): untyped =
```

Comment:
DRY: factor into a help, say `tests/lexer/mlexerutils.nim` instead of copy pasting

Reply:
done.